### ISSUE TITLE
CopyQ Crap files

### ISSUE THREAD

**Issue by Shakil-Shahadat** [MoscaDotTo/Winapp2#826]
```
[CopyQ *]
LangSecRef=3021
DetectFile=%AppData%\copyq
FileKey1=%AppData%\copyq|*.bak
FileKey2=%AppData%\copyq\copyq|copyq.log
FileKey3=%ProgramFiles%\CopyQ|AUTHORS;LICENSE;README.md
```
Note, the last entry will delete files that none should need.

**Comment by APMichael** [comment]
> 
> 
> ```
> FileKey3=%ProgramFiles%\CopyQ|AUTHORS;LICENSE;README.md
> ```
> Note, the last entry will delete files that none should need.

Please do not delete files like Authors, Changelog, Help, License and Readme. These are part of the application and not temporary "garbage".

Feel free to make an "License Text Files" entry in the Winapp3.ini for this.

**Comment by Shakil-Shahadat** [comment]
Update
```
[CopyQ *]
LangSecRef=3021
DetectFile=%AppData%\copyq
FileKey1=%AppData%\copyq|*.bak
FileKey2=%AppData%\copyq\copyq|copyq.log
```

**Comment by APMichael** [comment]
Thank you for the quick reply.

By the way, you are welcome to post several new or modified entries in one issue report. It is not necessary to open a separate report for each one.

**Comment by APMichael** [comment]
Included in #824.
